src/Assets/pageTitleTags.ts

```diff
@@ -0,0 +1,17 @@
+const pageTitleTags: Record<string, string> = {
```

Comment:
With Records, you want to include all of the keys as a union type. You could also just let the type be inferred because this is a constant.
Ex:
```
type StepName = 'step-1' | 'step-2' | ...
const pageTitleTags: Record<StepName, string> = {
```